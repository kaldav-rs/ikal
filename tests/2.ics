BEGIN:VCALENDAR
VERSION:2.0
PRODID:ownCloud Calendar
CALSCALE:GREGORIAN
BEGIN:VEVENT
UID:7e27d9fc69
DTSTAMP:20150717T101959Z
CREATED:20150717T101959Z
LAST-MODIFIED:20150717T101959Z
SUMMARY:Garage
DTSTART;VALUE=DATE:20150722
DTEND;VALUE=DATE:20150723
LOCATION:
DESCRIPTION:
CATEGORIES:
END:VEVENT
END:VCALENDAR
