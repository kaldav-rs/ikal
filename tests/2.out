Ok(
    VCalendar {
        prodid: "ownCloud Calendar",
        version: "2.0",
        calscale: Some(
            "GREGORIAN",
        ),
        method: None,
        content: Event(
            VEvent {
                dtstamp: 2015-07-17T12:19:59+02:00,
                uid: "7e27d9fc69",
                dt_start: 2015-07-22T00:00:00+02:00,
                dt_end: 2015-07-23T00:00:00+02:00,
                class: None,
                created: Some(
                    2015-07-17T12:19:59+02:00,
                ),
                description: Some(
                    "",
                ),
                geo: None,
                last_modified: Some(
                    2015-07-17T12:19:59+02:00,
                ),
                location: Some(
                    "",
                ),
                organizer: None,
                priority: None,
                seq: None,
                status: None,
                summary: Some(
                    "Garage",
                ),
                transp: None,
                url: None,
                recurid: None,
                rrule: None,
                attach: [],
                attendee: [],
                categories: [
                    "",
                ],
                comment: [],
                contact: [],
                exdate: [],
                rstatus: [],
                related: [],
                resources: [],
                rdate: [],
                x_prop: {},
                iana_prop: {},
            },
        ),
    },
)
